routes/server.go

```diff
@@ -1675,6 +1676,13 @@ func (fes *APIServer) NewRouter() *muxtrace.Router {
 			fes.AdminGetMempoolStats,
 			AdminAccess,
 		},
+		{
+			"AdminUpdateViewNumber",
+			[]string{"POST", "OPTIONS"},
+			RoutePathAdminUpdateViewNumber,
+			fes.AdminUpdateViewNumber,
+			SuperAdminAccess,
```

Comment:
Cool. I think adding this is sufficient to gate the endpoint on admin JWT access. We can check it once it's deployed somewhere.